src/store/reducers.js

```diff
@@ -26,6 +27,7 @@ export function makeRootReducer(asyncReducers: ?Reducers): Reducer<any, any> {
     wallets,
     walletsCreate,
     digitalAssets,
+    transactions,
```

Comment:
if so, please remove it's injecting inside transactions routes

Reply:
With clean installation application crashes without it

Reply:
Actually we have broken solution in develop